src/multibody/joint/joint-composite.hpp

```diff
@@ -200,9 +200,11 @@ namespace pinocchio
     /// \param jmodel Model of the joint to add.
     /// \param placement Placement of the joint relatively to its predecessor.
     ///
+    /// \return A reference to this JointModelComposite itself
```

Comment:
`A reference to  *this` is the way Eigen write their documentation. I think we should follow their example.

Reply:
Thanks for the suggestion